tests/TestCase/Model/Filter/ValueTest.php

```diff
@@ -46,6 +46,25 @@ public function testSkipProcess()
     /**
      * @return void
      */
+    public function testProcessWithoutField()
+    {
+        $articles = TableRegistry::get('Articles');
+        $manager = new Manager($articles);
+        $filter = new Value('title', $manager, [
+            'field' => []
```

Comment:
Shouldn't a filter configured without any field be treated as dev error and an exception be thrown?  I don't see a use case where empty field list would make sense.